examples/llm/inference/inference.py

```diff
@@ -111,9 +38,47 @@ def get_mosaicgpt_inference_model(checkpoint_yaml_path: str, tokenizer):
 
     model.load_state_dict(checkpoint['state']['model'], strict=True)
 
-    model.cuda()
-    model.eval()
+    model.tokenizer.pad_token_id = 50256
```

Comment:
maybe modify this to only set it if the tokenizer doesn't already have a pad token id? we may use a different tokenizer in the future that already has the pad token set.

Reply:
Fixed. Is there anything specific about 50256 or you just picked the last token?

Reply:
Just wondering how vocab_size would affect this.

Reply:
50256 is the eos token in the gpt tokenizer. thats it

Reply:
maybe just set it to tokenizer.eos_token_id instead of 50256, and emit a warning about doing that. cause if the vocab is smaller than 50256, it won't work

Reply:
Done